keychain/src/components/Transaction.tsx

```diff
@@ -59,15 +61,19 @@ export const Transaction = ({
       </HStack>
       <Spacer />
       <HStack spacing="15px">
-        <HStack color="gray.200" spacing="5px">
-          <StarknetIcon boxSize="14px" />
-          <Text color="inherit" fontSize="13px">
-            {chainId === constants.StarknetChainId.MAINNET
-              ? "Mainnet"
-              : "Testnet"}
-          </Text>
-        </HStack>
-        <Divider orientation="vertical" bgColor="gray.500" h="30px" />
+        {showChainId && (
```

Comment:
how come this is conditional?

Reply:
Where else are we using it? Shouldn't we always show the chain id?

Reply:
The transactions page doesn't show the chain ID. It's at the top. All transactions are the same chain. So no need to display the chain ID again (from calvins designs at least)